src/status_im/native_module/impl/module.cljs

```diff
@@ -54,9 +55,12 @@
 (defn init-jail []
   (when status
     (call-module
-     (fn []
-       (let [init-js (str js-res/status-js "I18n.locale = '" rn-dependencies/i18n.locale "';")]
-         (.initJail status init-js #(log/debug "jail initialized")))))))
+      (fn []
+        (let [init-js (str js-res/status-js "I18n.locale = '" rn-dependencies/i18n.locale "'; ")
+              init-js' (if config/jsc-enabled?
+                         (str init-js js-res/web3)
```

Comment:
So Is this the exact same version of web3 status-go injects? And is the order of injection the same (in case it matters)?

Reply:
Exactly here order doesn't matter. Yes, the same version, form our fork.